src/universal/components/AddToGitHubMenuItem.tsx

```diff
@@ -24,7 +24,7 @@ const AddToGitHubMenuItem = forwardRef((props: Props, ref) => {
       label={
         <MenuItemLabel>
           <MenuItemComponentAvatar>
-            <GitHubSVG />
+            <GitHubSVG18 />
```

Comment:
Why are we constraining a DOM SVG to a particular size? We should instead style the svg like any other element